frodo2-logger/build.gradle

```diff
@@ -1,30 +1,26 @@
-buildscript {
-  repositories {
-    jcenter()
-    maven {
-      url  "http://dl.bintray.com/android10/maven"
-    }
-    mavenLocal()
-  }
-  dependencies {
-    classpath "org.aspectj:aspectjtools:$project.aspectjVersion"
-  }
-}
+apply plugin: 'java'
+apply plugin: 'maven'
 
-repositories {
-  maven {
-    url  "http://dl.bintray.com/android10/maven"
-  }
-  mavenLocal()
-}
+targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
-  compile "org.jetbrains:annotations:$project.annotationsVersion"
-  compile "org.aspectj:aspectjrt:$project.aspectjVersion"
-  compile "io.reactivex.rxjava2:rxjava:$project.rxJavaVersion"
-  compile project(':frodo2-api')
+  implementation "org.jetbrains:annotations:$project.annotationsVersion"
```

Comment:
Out of curiosity, this was renamed right?

Reply:
What do you mean by renamed?

Reply:
using implementation rather than compile